scripts/launch.sh

```diff
@@ -36,6 +36,11 @@ if [[ "$TPS" != "" ]]; then
   panesCnt=$((n+1))
 fi
 
+# System start time in seconds (time since epoch).
+system_start=`date +%s`
+
+echo "Using system start time "$system_start
```

Comment:
Well, I don't know how hard it is too sum numbers in bash. If it's hard, just ignore, it shouldn't be important. If it's easy, I would add 1 or 2 sec.